Content.Shared/Species/Components/ReformComponent.cs

```diff
@@ -0,0 +1,48 @@
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Species.Components;
+
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+public sealed partial class ReformComponent : Component
+{
+    /// <summary>
+    /// The action to use.
+    /// </summary>
+    [DataField(required: true)]
+    public EntProtoId? ActionPrototype;
+
+    [DataField, AutoNetworkedField] 
+    public EntityUid? ActionEntity;
+
+    /// <summary>
+    /// How long it will take to reform
+    /// </summary>
+    [DataField("reformTime", required: true)]
```

Comment:
```suggestion
    [DataField(required: true)]
```
etc for the other ones not gonna spam 1000 reviews